### ISSUE TITLE
Code link has twice the hover delay of a normal link

### ISSUE THREAD

**Issue by hSaria** [squidfunk/mkdocs-material#1713]
__I checked that...__

- [X] ... the documentation does not mention anything about my problem
- [X] ... the problem doesn't occur with the default MkDocs template
- [X] ... the problem is not in any of my customizations (CSS, JS, template)
- [X] ... there are no open or closed issues that are related to my problem

## Description

When hovering over a link that includes code (i.e. surrounded part of the link with `` ` ``), the delay for changing the foreground color is inconsistent.

### Expected behavior

The hover delay of a link with code should match a link without one (normal link).

### Actual behavior

Code in a link has 500ms hover delay while normal (non-code) parts of the link have 250ms. It is most noticeable when a link has both.

### Steps to reproduce the bug

1. In a page, create a link like so:
   ```
   [Normal `code`](http://example.net)
   ```
2. Browse to the page and hover over the link (notice how `code` takes longer to change color)

Examples:
* [mkdocs-material docs](https://squidfunk.github.io/mkdocs-material/getting-started/#tabs): The ` mkdocs.yml` link
* [ChromaTerm docs](https://chromaterm.readthedocs.io/#api): The link in the note

### Package versions

* Python: 3.7.7
* MkDocs: 1.1.2
* Material: 5.2.0

### Project configuration

> No special configuration required

``` yaml
site_name: Example
site_url: https://example.net/
site_author: Example
repo_url: https://github.com/example/example/

theme: material

nav:
  - Example: 'index.md'
```

### System information

* OS: macOS Catalina 10.15.4
* Browser: Safari 13.1

**Comment by squidfunk** [comment]
That's the result of the transitions which are applied on all elements so the theme blends smoothly. I'll guess we'll just revert this behavior, as it causes these kinds of problems.

**Comment by squidfunk** [comment]
Should be fixed by a8fc0c5e.

**Comment by squidfunk** [comment]
Released as part of 5.2.1.
